server/controllers/users/profile.js

```diff
@@ -61,3 +62,32 @@ export const me = async function(req, res) {
   const user = await User.findById(req.session.passport.user)
   return res.json(user)
 }
+
+/**
+ * Make User an Admin
+ */
+export const promote = async function(req, res) {
+  const user = await User.findById(req.params.userId)
+  if (includes(user.roles, ADMIN_ROLE))
+    throw new BadRequestError('User already an admin')
+
+  user.roles.push(ADMIN_ROLE)
+  await user.save()
+  return res.json(user)
+}
+
+/**
+ * Take Admin priveleges away from User
+ */
+export const demote = async function(req, res) {
+  if (req.params.userId === req.user._id)
```

Comment:
`req.params.userId` is a string but `req.user._id` is a number

Reply:
Oh right. I forgot about that. Fixed.